AdminWebsite/AdminWebsite.UnitTests/Services/UserAccountServiceTests.cs

```diff
@@ -111,6 +116,21 @@ public async Task Should_add_JOH_role_to_JOH_user_group()
                 Times.Once);
         }
 
+        [Test]
+        public async Task Should_add_StaffMember_role_to_Staff_Member_user_group()
```

Comment:
Consider using Pascal case for test method names, eg:
ShouldAddStaffMemberRoleToStaffMemberUserGroup, is maybe a  little less cognitive strain for the reviewer. 

Also consider: 
NameOfMethodUnderTest_AddStaffMemberRoleToStaffMemberUserGroup to add some context.

Reply:
That´s done now